muxinc/mux-stats-sdk-avplayer
# Adds support for custom dimensions to MUXSDKStats

## Zube
https://zube.io/muxinc/mux/c/18768

## Overview

Custom dimensions can now be set on monitor or on videoChange just like other types of customer metadata.

## Dependencies
* https://github.com/muxinc/mux-stats-sdk-objc/pull/117
* https://github.com/muxinc/stats-sdk-objc/pull/67

## QA
Check out the custom dimensions in the [views generated by our UI tests](https://dashboard.mux.com/organizations/ffhei0/environments/k0vjj8/views). 😄   

* Custom dimension `c1` now tags the test case that the view was generated for which makes it easy to filter for all the views that contained ads vs views for normal playback (for example).
* Custom dimension `c2` is overridden by the test case that updates custom dimensions. 



![Screen Shot 2021-08-21 at 1 27 58 AM](https://user-images.githubusercontent.com/1444681/130311604-dc19180b-e19b-4166-8546-f9ca35b34c5c.png)
![Screen Shot 2021-08-21 at 1 28 14 AM](https://user-images.githubusercontent.com/1444681/130311605-7d9a0cf9-47c9-471c-8848-6b2b356779c9.png)
![Screen Shot 2021-08-21 at 1 28 23 AM](https://user-images.githubusercontent.com/1444681/130311606-c1117a7d-0949-437a-88cb-bd337de6d916.png)
![Screen Shot 2021-08-21 at 1 29 09 AM](https://user-images.githubusercontent.com/1444681/130311607-b1983a50-c89d-4ed9-8f40-ef24e9bf46e3.png)
